src/EnergyPlus/DesiccantDehumidifiers.cc

```diff
@@ -188,6 +188,8 @@ namespace DesiccantDehumidifiers {
 	int NumSolidDesicDehums; // number of solid desiccant dehumidifiers
 	int NumGenericDesicDehums; // number of generic desiccant dehumidifiers
 	Real64 TempSteamIn( 100.0 ); // steam coil steam inlet temperature
+	static bool GetInputFlag( true ); // First time, input is "gotten"
+	static bool MyOneTimeFlag( true );
```

Comment:
Echoing @mbadams5 comments in https://github.com/NREL/EnergyPlus/commit/d0c373af1b8ce68bd8fd91bc30b4909f09d1c519 , these flags that should be in an anonymous namespace.  Follow the example [here in HVACManager.cc](https://github.com/NREL/EnergyPlus/blob/b37f69d17bfd8e2075b7125d0448f336ad0200c9/src/EnergyPlus/HVACManager.cc#L209-L218).